codegen/types.go

```diff
@@ -143,7 +143,7 @@ func goTypeDefObject(o design.Object) string {
 	var ss []string
 	ss = append(ss, "struct {")
 	o.WalkAttributes(func(name string, at *design.AttributeExpr) error {
-		ss = append(ss, fmt.Sprintf("%s %s", name, GoTypeName(at.Type)))
+		ss = append(ss, fmt.Sprintf("\t%s %s", name, GoTypeName(at.Type)))
```

Comment:
Ah wait, ignore my first comment. I missed the part where you were building a structure definition here.
Sorry for the confusion